run_unit_tests.sh

```diff
@@ -22,6 +22,7 @@ fi
 which airflow > /dev/null || python setup.py develop
 
 echo "Initializing the DB"
+rm $AIRFLOW_HOME/unittests.db
```

Comment:
I think it would be preferable to replace the initdb statement with a 
```
airflow resetdb
```

This would make the script slightly more concise and, more importantly, it would avoid repeating in one more place the `$AIRFLOW_HOME/unittests.db` location of the test DB. 

I would also make the reste more generic since the `rm` statement above is specific to sqlite whereas mysql and postresql are other possibilities. The Travis build executes the unit tests on all 3 DB backends.